### PR TITLE
fix card color

### PR DESCRIPTION
**Pull request by stormcloud266**
update the passkey credential card bg color

### CODE REVIEW

**Comment by stormcloud266** [comment]
* **#425** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/425?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `dev`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @stormcloud266 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by 0xBigBoss — APPROVED** [review]

File: packages/app/features/account/settings/backup/index.tsx [hunk 1]
```diff
@@ -157,7 +157,7 @@ const SendAccountCredentials = ({
       w={'100%'}
       gap={'$size.1.5'}
       p={'$size.1.5'}
-      $theme-dark={{ backgroundColor: '$darkest' }}
+      $theme-dark={{ backgroundColor: '$black' }}
       $theme-light={{ backgroundColor: '$gray2Light' }}
```
Comment (youngkidwarrior) [inline packages/app/features/account/settings/backup/index.tsx:161]:
```
$theme-dark={{ backgroundColor: '$darkest' }}
$theme-light={{ backgroundColor: '$gray2Light' }}
```
There should be a radix color for this now. I think its this
```suggestion
      bc={"$color1"}
```

**Review by youngkidwarrior — DISMISSED** [review]

File: packages/app/features/account/settings/backup/index.tsx (hunk 1 above)
Comment (stormcloud266) [reply packages/app/features/account/settings/backup/index.tsx]: Nice yeah that worked, that's super handy to know.

Comment (youngkidwarrior) [reply packages/app/features/account/settings/backup/index.tsx]: Yeah major QOL upgrade

**Review by 0xBigBoss — APPROVED** [review]

**Comment by stormcloud266** [comment]
### Merge activity

* **May 20, 3:54 PM EDT**: @stormcloud266 merged this pull request with [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/425).

**Pull request merged**